pcsx2/Docs/Configuration_Guide/Configuration_Guide.md

```diff
@@ -1,6 +1,8 @@
-# PCSX2 Configuration: Windows
+# PCSX2 First Time Setup and Configuration
 This guide will walk through the initial setup process of PCSX2 and the basics of configuration.
 
+*Note: While this document is using Windows screenshots, you can still follow this guide for Linux.*
```

Comment:
I don't really think this line is necessary. It just pollutes reading (and/or makes linux sound as subpar)